camel-openmrs-fhir/src/main/java/org/openmrs/eip/fhir/routes/resources/SupplyRequestRouter.java

```diff
@@ -0,0 +1,70 @@
+package org.openmrs.eip.fhir.routes.resources;
+
+import static java.util.Base64.getEncoder;
+
+import static org.openmrs.eip.fhir.Constants.HEADER_FHIR_EVENT_TYPE;
+import static org.openmrs.eip.fhir.Constants.PROP_EVENT_OPERATION;
+import static org.openmrs.eip.fhir.Constants.SUPPLY_REQUEST_ORDER_TYPE_UUID;
+
+import java.util.Collections;
+
+import org.apache.camel.LoggingLevel;
+import org.hl7.fhir.r4.model.Quantity;
+import org.hl7.fhir.r4.model.Reference;
+import org.hl7.fhir.r4.model.SupplyRequest;
+import org.openmrs.eip.fhir.FhirResource;
+import org.openmrs.eip.fhir.routes.resources.models.Order;
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@Component
+public class SupplyRequestRouter extends BaseFhirResourceRouter {
+	
+	private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Let's use Jackson Library provided by Camel for unmarshalling

Reply:
Yeah that was my initial implementation but that doesn't supports this annotation @JsonIgnoreProperties(ignoreUnknown = true) because of which I have to use the ObjectMapper

Reply:
Under the hood `camel-jackson` uses Object mapper from Jackson library. This should work, you need to enable that feature using the property https://camel.apache.org/components/4.8.x/dataformats/jackson-dataformat.html#_dataformat_option_enableFeatures

Reply:
Okay thanks for sharing.